src/components/Input/Search.js

```diff
@@ -94,15 +100,20 @@ export default class SearchInput extends React.Component {
     }));
 
   render() {
-    const { slim, invert, ...props } = this.props;
+    const { slim, invert, disabled, onClick, ...props } = this.props;
     return (
-      <Container slim={slim} invert={invert}>
+      <Container
+        slim={slim}
+        invert={invert}
+        onClick={e => disabled && onClick(e)}
```

Comment:
We want this `onClick` to only fire when this `this.props.disabled` equals `true`? Is the use case here to enable the input `onClick`?